codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -646,9 +647,10 @@ private LastHttpContent readTrailingHeaders(ByteBuf buffer) {
                         b.append(currentLastPos)
                          .append(lineTrimmed);
                         current.set(lastPos, b.toString());
-                    } else {
-                        // Content-Length, Transfer-Encoding, or Trailer
                     }
+                    //else {
+                    // Content-Length, Transfer-Encoding, or Trailer
```

Comment:
Knowing full well what I'm about to say will likely not have my intended effect, backfire and lead to me being viewed as contrary or difficult, as a user, follower, and occasional contributor to Netty, I feel it's worth mentioning these distractions (which happen quite frequently) just increase the cost of contributing. This comment has nothing to do with this PR. This discussion would have been better started on a separate issue.